docs/components/console/manage-clusters/resume-cluster.md

```diff
@@ -1,7 +1,17 @@
 ---
 id: resume-cluster
 title: Resume your cluster
-description: "Follow these instructions to resume your cluster"
+description: "You can resume your paused cluster at any time"
 ---
 
-A trial cluster can be paused automatically due to inactivity. To resume your cluster, navigate to the **Clusters** tab, click on the cluster name (e.g. Prod Cluster - Project A) from the list and in the cluster details, and click **Resume cluster**.
+If your cluster is paused, you can resume it at any time.
+
+To resume your paused cluster:
+
```

Comment:
This is the long way to achieve this goal.

It's a better UX to use the Web Modeler's deploy modal. There's a button that says "resume" cluster.

![image](https://github.com/user-attachments/assets/1f76442b-903f-4030-9ff4-cd68da3b8ce0)

Reply:
I have added in this detail to the Resume cluster page - thanks for highlighting 👍